tests/nominal/test_GroupRareLevelsTransformer.py

```diff
@@ -57,21 +57,30 @@ def test_weight_not_str_error(self):
         ):
             GroupRareLevelsTransformer(weight=2)
 
-    def test_record_rare_levels_not_str_error(self):
+    def test_record_rare_levels_not_bool_error(self):
         """Test that an exception is raised if record_rare_levels is not a bool."""
         with pytest.raises(
             ValueError,
             match="GroupRareLevelsTransformer: record_rare_levels must be a bool",
         ):
             GroupRareLevelsTransformer(record_rare_levels=2)
 
+    def test_encode_unseen_levels_not_bool_error(self):
+        """Test that an exception is raised if encode_unseen_levels is not a bool."""
+        with pytest.raises(
+            ValueError,
+            match="GroupRareLevelsTransformer: encode_unseen_levels must be a bool",
+        ):
+            GroupRareLevelsTransformer(encode_unseen_levels=2)
+
     def test_values_passed_in_init_set_to_attribute(self):
```

Comment:
Thought we deleted these tests but must have missed some.  Please delete this and `test_arguments`, `test_class_methods` throughout this test module. We are ditching implementation tests where possible as they create friction for no real value.